appserver/deployment/dol/src/main/java/com/sun/enterprise/deployment/annotation/introspection/ConstantPoolInfo.java

```diff
@@ -122,6 +122,17 @@ public boolean containsAnnotation(int constantPoolSize, final ByteBuffer buffer)
                     buffer.getShort();
                     buffer.getShort();
                     break;
+                case METHODHANDLE:
```

Comment:
It is enough to add the C2B2 Copyright line once per file after the license header line in the beginning of the file. For an examples, see https://github.com/payara/Payara/blob/master/appserver/pom.xml#L43